firmware/src/checkpoint.rs

```diff
@@ -60,6 +60,9 @@ pub enum CheckpointVariant {
 
     #[cbor(n(7))]
     Removed,
+
+    #[cbor(n(8))]
+    ShowMnemonic,
```

Comment:
I would add a field in here to save whether it's a "show mnemonic" command or an initial backup. I think we can reuse this checkpoint logic there as well, but when we reboot we need to distinguish which handle to resume

Reply:
Is a bool enough? Done in 8704eae59889020230f1fa96b0e982bd4628fb5b